behave/features/steps/fala_end_to_end_tests.py

```diff
@@ -177,3 +169,22 @@ def step_impl_translated(context, code_indicator, title_text_starts_with):
 
     assert updated_page == f"{code_indicator}"
     assert updated_title_gui.startswith(f"{title_text_starts_with}")
+
+
+@step("There are less results visible on the results page")
+def step_impl_fewer_results_returned(context):
+    total_results = context.helperfunc.find_by_xpath(
+        '//section/p[@class="govuk-body"]'
+    ).text.split()[0]
+    assert int(total_results) < context.results
```

Comment:
again, will it always be less? I assume the test case is set up so it will be?

Reply:
Yes. The purpose of the test is to first store the total results with no filter applied. This effectively returns all providers of every type. This number is stored as context.results.

This step is used after applying a filter where the new total results will always be less.

Reply:
@MishNajam if the provider only supplies information for one category from the filter then it would not be less but I take what you are saying to mean that this is a provider which fits more than one category.